DrivewayDentDeletion/Operators/cicd-webhook-triggers.yaml

```
@@ -12,485 +24,70 @@ spec:
-    - https://raw.githubusercontent.com/IBM/charts/master/repo/entitled/
-    - --namespace
-    - ace
-    - --set
-    - image.pullPolicy=Always,image.aceonly=image-registry.openshift-image-registry.svc:5000/ace/ace-ddd-api:$(params.imageTag),aceonly.replicaCount=1,license=accept,integrationServer.configurationSecret=ace-ddd-dev-creds
-    - --tls
-    - --install
-    - --debug
-    image: alpine/helm:2.12.3
-    env:
-      - name: KUBECONFIG
-        value: /app/kubeconfig
-      - name: HELM_TLS_CA_CERT
-        value: /app/helm/ca.pem
-      - name: HELM_TLS_CERT
-        value: /app/helm/cert.pem
-      - name: HELM_TLS_KEY
-        value: /app/helm/key.pem
-    volumeMounts:
-     - name: kubeconfig
-       mountPath: /app
-     - name: helm-tls
-       mountPath: /app/helm
-     - name: helm-repositories
-       mountPath: /tekton/home/.helm/repository
-    name: helm-deploy
-    resources: {}
-    securityContext:
-      runAsUser: 0
-  volumes:
-    - name: kubeconfig
-      secret:
-        secretName: cluster-kubeconfig
-    - name: helm-tls
-      secret:
-        secretName: task-helm-tls
-    - name: helm-repositories
-      secret:
-        secretName: task-helm-repositories
-
----
-
-apiVersion: tekton.dev/v1alpha1
-kind: Task
-metadata:
-  name: helm-deploy-task-ace-acme
-spec:
-  params:
-    - name: imageTag
+      description: The image name
+    - name: releaseName
       type: string
-      description: The image tag
-  steps:
-  - args:
-    - upgrade
-    - ace-ddd-acme-dev
-    - ibm-ace-server-icp4i-prod
-    - --repo
-    - https://raw.githubusercontent.com/IBM/charts/master/repo/entitled/
-    - --namespace
-    - ace
-    - --set
-    - image.pullPolicy=Always,image.aceonly=image-registry.openshift-image-registry.svc:5000/ace/ace-ddd-acme:$(params.imageTag),aceonly.replicaCount=2,license=accept,integrationServer.configurationSecret=ace-ddd-dev-creds
-    - --tls
-    - --install
-    - --debug
-    image: alpine/helm:2.12.3
-    env:
-      - name: KUBECONFIG
-        value: /app/kubeconfig
-      - name: HELM_TLS_CA_CERT
-        value: /app/helm/ca.pem
-      - name: HELM_TLS_CERT
-        value: /app/helm/cert.pem
-      - name: HELM_TLS_KEY
-        value: /app/helm/key.pem
-    volumeMounts:
-     - name: kubeconfig
-       mountPath: /app
-     - name: helm-tls
-       mountPath: /app/helm
-     - name: helm-repositories
-       mountPath: /tekton/home/.helm/repository
-    name: helm-deploy
-    resources: {}
-    securityContext:
-      runAsUser: 0
-  volumes:
-    - name: kubeconfig
-      secret:
-        secretName: cluster-kubeconfig
-    - name: helm-tls
-      secret:
-        secretName: task-helm-tls
-    - name: helm-repositories
-      secret:
-        secretName: task-helm-repositories
-
----
-
-apiVersion: tekton.dev/v1alpha1
-kind: Task
-metadata:
-  name: helm-deploy-task-ace-bernie
-spec:
-  params:
-    - name: imageTag
+      description: release name of queue manager deployment
```

Also used for ace, so maybe:
```suggestion
      description: release name the deployment
```

---

Done